### ISSUE TITLE
Find a way to get God log to STDOUT / STDERR instead of files

### ISSUE THREAD

**Issue by yivo** [openware/peatio#335]
One solution will be to symlink logs to virtual Linux devices for standard streams.

**Comment by shal** [comment]
I found [resource](http://godrb.com) for this issue.

__REDIRECTING STDOUT AND STDERR OF YOUR PROCESS__

By default, the __`STDOUT`__ stream for your process is redirected to `/dev/null`. To get access to this output, you can redirect the stream either to a file or to a command.

To redirect `STDOUT` to a file, set the log attribute to a file path. The file will be written in append mode and created if it does not exist.

```ruby
God.watch do |w|
  ...

  w.log = '/var/log/myprocess.log'

  ...
end
```
To redirect STDOUT to a command that will be run for you, set the log_cmd attribute to a command.
```ruby
God.watch do |w|
  ...

  w.log_cmd = '/usr/bin/logger'

  ...
end
```

By default, `STDERR` is redirected to `STDOUT`. You can redirect it to a file or a command just like `STDOUT` by setting the `err_log` or `err_log_cmd` attributes respectively.

**Comment by shal** [comment]
@yivo maybe we only should add `w.log_cmd = '/usr/bin/logger'` to `lib/daemons/daemons.god` file.

**Comment by yivo** [comment]
@ashanaakh no. This will break logs on dev machine.

**Comment by yivo** [comment]
They meant `/usr/bin/logger` is some custom script for logging. The solution may be to set log_cmd & err_log_cmd only for PROD env which both point to some script like `Rails.root.join('bin/god_logger.sh')`.

**Comment by yivo** [comment]
Implemented in #363.
